### PR TITLE
Package baseline

### PR DESCRIPTION
**Pull request by ericstj**
This enables the package baseline to live in the repo that builds the
packages, avoiding a roundtrip to buildtools every time we update
package versions.

I've updated the baseline with the latest changes to corefx package 
versions.

Requires: https://github.com/dotnet/buildtools/pull/646

### CODE REVIEW

**Comment by ericstj** [comment]
Removed WIP tag, this is ready to go.  @weshaggard can you have a look?

**Comment by ericstj** [comment]
@dotnet-bot please test Innerloop Windows_NT Debug Build and Test

**Comment by ericstj** [comment]
@dotnet-bot please test Innerloop Windows_NT Release Build and Test

**Comment by ericstj** [comment]
@dotnet-bot test  Innerloop Windows_NT Debug Build and Test

**Comment by ericstj** [comment]
@dotnet-bot test Innerloop Windows_NT Release Build and Test

File: Packaging.props [hunk 1]
```diff
@@ -12,6 +12,8 @@
     <PackagePlatform Condition="'$(PackagePlatform)' == 'amd64'">x64</PackagePlatform>
     <NativePackagePath>$(MSBuildThisFileDirectory)src/Native/pkg</NativePackagePath>
   </PropertyGroup>
+  
+  <Import Project="pkg/Microsoft.Private.PackageBaseline/Microsoft.Private.PackageBaseline.props" />
```
Comment (weshaggard) [inline Packaging.props]: What is your plan for consuming this in other repo's like WCF? I would like to make sure they don't have to do anything too crazy.

**Comment by weshaggard** [comment]
cc @StephenBonikowsky 

@ericstj do you plan on making a similar update in WCF?

**Comment by ericstj** [comment]
I've already done so: https://github.com/dotnet/wcf/pull/1050

File: Packaging.props (hunk 1 above)
Comment (ericstj) [inline Packaging.props]: See: https://github.com/dotnet/wcf/pull/1050

**Comment by weshaggard** [comment]
LGTM

**Comment by ericstj** [comment]
@dotnet-bot test Innerloop CentOS7.1 Debug Build and Test

**Comment by ericstj** [comment]
@dotnet-bot test  Innerloop OSX Debug Build and Test

**Comment by ericstj** [comment]
@dotnet-bot test Innerloop OSX Release Build and Test

**Comment by StephenBonikowsky** [comment]
Made a similar comment in dotnet/wcf#1050
It would be ideal if certain shared scripts could be unified in both our branches.
While it makes them uglier it also makes them easier to maintain.

**Comment by ericstj** [comment]
Refactored this to make the diff for packaging.props identical between the two branches.

**Comment by ericstj** [comment]
Further refactored to make better diff of dir.props between corefx and wcf

**Comment by StephenBonikowsky** [comment]
LGTM

**Comment by ericstj** [comment]
Windows debug failed due to https://github.com/dotnet/corefx/issues/6255
@dotnet-bot test Innerloop Windows_NT Debug Build and Test

**Comment by ericstj** [comment]
@dotnet-bot test  Innerloop OSX Release Build and Test

**Comment by ericstj** [comment]
OSX build has passed prior to rebase and latest failure is due to failing network test.  Merging now.

**Pull request merged**